src/egon/data/datasets/scenario_parameters/parameters.py

```diff
@@ -314,27 +314,32 @@ def electricity(scenario):
         # marginal cost can include fuel, C02 and operation and maintenance costs
         parameters["marginal_cost"] = {
             "oil": global_settings(scenario)["fuel_costs"]["oil"]
+            / read_costs(costs, "oil", "efficiency")
             + read_costs(costs, "oil", "VOM")
             + global_settings(scenario)["co2_costs"]
             * global_settings(scenario)["co2_emissions"]["oil"],
             "other_non_renewable": global_settings(scenario)["fuel_costs"][
                 "gas"
-            ]
+            ] / read_costs(costs, "OCGT", "efficiency")
             + global_settings(scenario)["co2_costs"]
             * global_settings(scenario)["co2_emissions"][
                 "other_non_renewable"
             ],
             "lignite": global_settings(scenario)["fuel_costs"]["lignite"]
+            / read_costs(costs, "lignite", "efficiency")
             + read_costs(costs, "lignite", "VOM")
             + global_settings(scenario)["co2_costs"]
             * global_settings(scenario)["co2_emissions"]["lignite"],
```

Comment:
The Co2 costs were not changed, which would mean that these costs have been in EUR / MWh_el. Is that correct?

Reply:
I thought so but they are actually based on the CO2 emission of the source, so the efficiency must be considered. Already corrected in the last commit.